src/App.tsx

```diff
@@ -1,34 +1,121 @@
-/* eslint-disable max-len */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 
 import { TodoList } from './components/TodoList';
 import { TodoFilter } from './components/TodoFilter';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
+import { Todo } from './types/Todo';
+import { getTodos } from './api';
+
+enum SortType {
+  ALL = 'all',
+  ACTIVE = 'active',
+  COMPLETED = 'completed',
+}
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [visibleTodos, setVisibleTodos] = useState(todos);
+  const [loading, setLoading] = useState(false);
+  const [userId, setUserId] = useState(0);
+  const [selectedTodo, setSelectedTodo] = useState(0);
+  const [selectedUser, setSelectedUser] = useState(false);
+  const [query, setQuery] = useState('');
+  const [filteredBy, setFilteredBy] = useState<string>(SortType.ALL);
+
+  useEffect(() => {
+    const loadTodos = async () => {
+      const todosFromServer = await getTodos();
+
+      setLoading(true);
+      setTodos(todosFromServer);
+      setVisibleTodos(todosFromServer);
+    };
+
+    loadTodos();
+    setLoading(false);
+  }, []);
+
+  const changeFilteredBy = (filterType: string) => {
+    setFilteredBy(filterType);
+  };
+
+  const changeQuery = (input: string) => {
```

Comment:
```suggestion
  const handleQueryChange = (input: string) => {
```
https://jaketrent.com/post/naming-event-handlers-react

Reply:
make sure to fix it all over your code